opentelemetry-api/src/opentelemetry/util/types.py

```diff
@@ -56,3 +56,18 @@
     ],
     ...,
 ]
+
+
+@dataclass
+class MetricsHistogramAdvisory:
+    explicit_bucket_boundaries: Optional[Sequence[float]] = None
+
+
+@dataclass
+class MetricsCommonAdvisory:
+    pass
+
+
+MetricsInstrumentAdvisory = Union[
```

Comment:
Likewise if these are private can we do an `_` prefix for now or hide them in the `_internal` package?

Reply:
Done in 5cd650ae40f3b92046d34c8d789a3dfa4d52697b